apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx

```diff
@@ -35,6 +36,7 @@ async function Portal() {
 			<VerticalTimeline status={status as PortalStatus} />
 			<Message status={status as PortalStatus} />
 			{isAccepted && <SignWaiver status={status as PortalStatus} />}
+			{isAccepted && <ConfirmAttendance status={status} />}
```

Comment:
Issue: We'll want to show this only when the user has completed the waiver, so the status can only be `PortalStatus.waived` or `PortalStatus.confirmed` in this case.